include/openssl/ssl.h.in

```diff
@@ -329,6 +329,10 @@ typedef int (*SSL_async_callback_fn)(SSL *s, void *arg);
 # define SSL_OP_ENABLE_KTLS                              0x00000008U
 
 # define SSL_OP_TLSEXT_PADDING                           0x00000010U
+
+/* Disable RFC8879 certificate compression */
+# define SSL_OP_NO_CERTIFICATE_COMPRESSION               0x00000020U
```

Comment:
OpenSSL flags are very limited. Maybe it will be better to use an additional function to store enabled compression algorithms in SSL/SSL_CTX->ext?

Reply:
The SSL_OP flags tie in with the configuration file nicely; that is why they are there. In addition, a number of flags become available in 3.0 due to deprecation of old features.

Reply:
Maybe add something like **SSL_OP_CIPHER_SERVER_PREFERENCE** to force the client to use brotli in case of zlib first in the list but brotli or zstd on the second position?

Reply:
An API to specify the the preference crypto order would probably work better. Your suggestion for promoting a single algorithm is very limited.